pipeline/inputs/node-exporter-metrics.md

```diff
@@ -33,6 +33,7 @@ This plugin is currently only supported on Linux based operating systems\
 | collector.loadavg.scrape\_interval   | The rate in seconds at which loadavg metrics are collected from the host operating system. If a value greater than 0 is used then it overrides the global default otherwise the global default is used. | 0 seconds |
 | collector.vmstat.scrape\_interval   | The rate in seconds at which vmstat metrics are collected from the host operating system. If a value greater than 0 is used then it overrides the global default otherwise the global default is used. | 0 seconds |
 | collector.filefd.scrape\_interval   | The rate in seconds at which filefd metrics are collected from the host operating system. If a value greater than 0 is used then it overrides the global default otherwise the global default is used. | 0 seconds |
+| collector.processes.scrape\_interval | The rate in seconds at which processes metrics are collected from the host operating system. If a value greater than 0 is used then it overrides the global default otherwise the global default is used. | 0 seconds |
```

Comment:
I would use the singular `process` in these cases.

It might be good to explain what process metrics are available.

Reply:
Yeah, sorry I meant in the description.

Reply:
Ohh, OK. This could be working as mentioning "process level of metrics".

Reply:
Fixed in https://github.com/fluent/fluent-bit-docs/pull/1184/commits/922a1fa936af25c93736afceb4e142ad3efb238b.